.vscode/settings.json

```diff
@@ -2,5 +2,8 @@
     "files.eol": "\n",
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
+    "files.associations": {
+        "*.yml": "ansible"
+    },
```

Comment:
this could get annoying, for me anyway the plugin is already too aggressive in assuming every YAML file is ansible and I keep having to add overrides to certain paths. We might want to consider .gitignoring the entire file, or see if there's a way to suggest things maybe? I know there's a way to (for example) suggest extensions.. forget the syntax or filename..

Reply:
Good call, will address in the upcoming branch to fix the sanity test